pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -38,8 +38,14 @@ service RuntimeService {
     // ContainerStatus returns status of the container.
     rpc ContainerStatus(ContainerStatusRequest) returns (ContainerStatusResponse) {}
 
-    // Exec executes the command in the container.
-    rpc Exec(stream ExecRequest) returns (stream ExecResponse) {}
+    // Exec prepares a streaming endpoint to execute a command in the container.
+    rpc Exec(ExecRequest) returns (ExecResponse) {}
+    // ExecSync runs a command in a container synchonously.
```

Comment:
typo: s/synchonously/synchronously

nit (optional): swap the order of `Exec` and `ExecSync` so that all streaming requests are grouped together